src/index.js

```diff
@@ -151,13 +151,14 @@ class Plugin {
 	getLogMetricCF(alarm, functionName, normalizedFunctionName){
 		var output = {};
 		if (alarm.pattern) {
-			const logMetricCFRefBase = this.naming.getLogMetricCFRef(normalizedFunctionName,alarm.name);
+			const stackName = _.upperFirst(_.camelCase(this.serverless.getProvider('aws').naming.getStackName()));
+			const logMetricCFRefBase = this.naming.getLogMetricCFRef(normalizedFunctionName,stackName,alarm.name);
```

Comment:
You don't need to add stack name to CF resources. It'll do it for you automatically.